report/src/discussion/discussion.tex

```diff
@@ -1,37 +1,40 @@
+As mentioned in Section \ref{sec:ChoiceOfTechnique}, the goal of the project was not to achieve high accuracy and precision on the exact positioning of people in a room. 
+The goal was to investigate whether we could detect the presence of people in a meeting with reasonable certainty.
+The results show that while we can detect with high precision that the room is in use, the precision of the classification of when the room is unused is more dubious. 
+However, for the purpose of this system, this is not overly problematic. 
+When the occupants exits and leaves the room after a meeting, the between the room and the phone increases, and the classification naturally skews towards \textit{outside}.
+Furthermore, these findings align with what similar research has concluded; that using kNN is sufficiently accurate, and that complex machine learning is not necessary to obtain useful results.\cite{ble_kneares_neural}
 
-However, it is worth noticing that for measuring points \textit{B} and \textit{J} in Figure \ref{fig:experiment_room}, a large uncertainty in classification is observed.
+It is worth noticing that for measuring points \textit{B} and \textit{J} in Figure \ref{fig:experiment_room}, a large uncertainty in classification is observed.
 We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it.
 This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as \textit{inside} and some as \textit{outside}, therefore giving uncertain classifications.
 
-A limitation with the presented approach is that it requires employees to carry a device capable of running the application in the background.
-This makes the approach unsuitable for organizations that do not provide their employees with such devices.
+A limitation of the presented approach is that it requires employees to carry a device capable of running the application in the background.
+Therefore, the approach can be unsuitable for organizations that do not provide their employees with such devices.
 It is a tradeoff between convenience and accuracy.
-Proximity detection, for example, would have been more convinient, since it would only require employees to carry a Bluetooth-enabled devices (such as a mobile phone or Bluetooth tag). However, this method is far less accurate, as discussed in Section~\ref{sec:proximity_detection}.
+Proximity detection, for example, would have been more convenient, since it would only require employees to carry a Bluetooth-enabled device (such as a mobile phone or Bluetooth tag). However, this method is far less accurate, as discussed in Section~\ref{sec:proximity_detection}.
```

Comment:
```suggestion
In this case, proximity detection could have been more convenient, as it would only require employees to carry a Bluetooth-enabled device, such as a mobile phone or Bluetooth tag.
However, this method is far less accurate, as discussed in Section~\ref{sec:proximity_detection}.
```